src/main/java/com/spring/studentmanagement/security/SecurityConfig.java

```diff
@@ -43,7 +43,7 @@ public AppAuthManager authenticationManager() {
     public FilterRegistrationBean<AuthFilter> authFilterFilterRegistrationBean() {
         final FilterRegistrationBean<AuthFilter> registrationBean = new FilterRegistrationBean<>();
         final String[] urlPatterns = {
-                "/users/*",
+                "/users/*", "/courses"
```

Comment:
"/courses/*" 

* - means that we secure every single endpoint starting with /courses. 
Good job!  @gonciarstef  😊